src/buckets/components/BucketCardMeta.tsx

```diff
@@ -56,9 +56,14 @@ const BucketCardMeta: FC<Props> = ({bucket, notify}) => {
     <span data-testid="bucket-schemaType"> {schemaLabel} </span>
   )
 
-  const bucketInfo = CLOUD
-    ? [persistentBucketMeta, schemaBlock]
-    : [persistentBucketMeta]
+  const bucketInfo = CLOUD ? (
+    <>
+      {persistentBucketMeta}
+      {schemaBlock}
+    </>
+  ) : (
```

Comment:
has this been tested when cloud is negative?
just that the second clause is not in curly braces.

want to be extra sure

Reply:
@jrenee42 

It _should_ be fine, since it's doing a variable assignment and the assigned variable (`bucketInfo`) is rendered using curly braces.

But this is a good point. I've actually tweaked the 'solution' to be less of change. I just added `key` props to the two components, rather than changing how they're rendered. Fixes the problems and keeps things similar to the way they were originally authored.